src/components/navbar-header-panels/side-menu-panel.component.tsx

```diff
@@ -1,15 +1,31 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { ExtensionSlot } from '@openmrs/esm-framework';
 import { SideNav } from 'carbon-components-react/es/components/UIShell';
 import { SideNavProps } from 'carbon-components-react';
```

Comment:
You could make it an `import type` just to be sure the module is really discarded (usually it is anyway).